cap_anndata/backed_df.py

```diff
@@ -31,10 +35,64 @@ def __setitem__(self, key, value) -> None:
         return super().__setitem__(key, value)
 
     @classmethod
-    def from_df(cls, df: pd.DataFrame, column_order: List[str] = None):
+    def from_df(cls, df: pd.DataFrame, column_order: List[str] = None) -> Self:
         if column_order is None:
             column_order = df.columns.to_numpy()
 
         new_inst = cls(df)
         new_inst.column_order = column_order
         return new_inst
+
+    def join(self, other: Any, on=None, how="left", lsuffix="", rsuffix="", sort=False):
```

Comment:
please change to `left_suffix` and `right_suffix` on our side

Reply:
replaced with **kwargs